website/server/controllers/api-v3/groups.js

```diff
@@ -127,7 +127,7 @@ api.getGroups = {
     if (validationErrors) throw validationErrors;
 
     let types = req.query.type.split(',');
-    let groupFields = basicGroupFields.concat(' description memberCount balance');
+    let groupFields = basicGroupFields.concat(' description memberCount balance purchased');
```

Comment:
Do you mean, shouldn't be shown? I do need to check if there is an active subscription though - for all users.